sources/Debate-frontend/src/app/debate/add-update-debate/add-update-debate.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="content">
+    <h1>Dodaj debatę</h1>
+
+    <form class="debate-form" #debateForm="ngForm" (ngSubmit)="onSubmit()">
+
+        <mat-form-field hintLabel="5-50 znaków" class="form-field">
+            <mat-label>Tytuł debaty</mat-label>
+            <input matInput maxlength="50" minlength="5" required type="text" name="title" placeholder="Tytuł..." [(ngModel)]="model.title">
+            <mat-error *ngIf="model.title.length < 5">Tytuł musi mieć co najmniej 5 znaków</mat-error>
```

Comment:
```suggestion
            <mat-error *ngIf="title.errors?.minlength">Tytuł musi mieć co najmniej 5 znaków</mat-error>
```

Use Angular forms provided errors rather than manual check